js/ccf-app/src/converters.ts

```diff
@@ -56,6 +63,25 @@ function checkString(val: any) {
   }
 }
 
+function checkJsonSafe(val: any) {
+  // Hard to be exhaustive, but throw errors for any Map or Date elements found
```

Comment:
It seems like it would be safer to allow known safe types (Number, String, Object, Array etc)?